### PR TITLE
Rename Goldschläger to Gildlager

### LINKED ISSUE #10500: Remove references to Goldschläger
It's trademarked, so it would be better to just not refer to it directly. The sprite can stay the same. The "gimmick" with Goldschläger is that it has gold flakes floating in it, so that should be part of the new name and/or description.

### PR DESCRIPTION
**Pull request by UbaserB**
## About the PR

Resolves #10500. Also adds an extra stock in the booze-o-mat so it matches champagne.

### CODE REVIEW

**Review by EmoGarbage404 — CHANGES_REQUESTED** [review]
needs migration.yml entries for `DrinkGoldschlagerBottleFull` and `DrinkGoldschlagerGlass`

**Comment by Hmeister-fake** [comment]
Can it keep the ä? It makes it look more unique

**Comment by UbaserB** [comment]
> Can it keep the ä? It makes it look more unique

it doesnt actually have an ä ingame

**Comment by Emisse** [comment]
conflict

**Comment by UbaserB** [comment]
> conflict

shut up i cant fix it!! This PR might as well be closed since literally no one can help me

**Comment by IamVelcroboy** [comment]
I just want to know if you even considered Goldschlonger at all....

**Comment by GreyMario** [comment]
> I just want to know if you even considered Goldschlonger at all....

dick joke ban he

File: Resources/migration.yml [hunk 1]
```diff
@@ -123,3 +123,8 @@ WarpPointBeaconService: null
 # 2023-12-30
 ClothingEyesGlassesCosmeticSunglasses: ClothingEyesGlassesSunglasses
 SpawnPointAssistant: SpawnPointPassenger
+
+# 2024-01-05
+Goldschlager: Gildlager
```
Comment (metalgearsloth) [inline Resources/migration.yml:128]: Needs to be an entity prototype.

**Pull request merged**